### DISCUSSION TITLE
Blueprint to create YAML array of strings

### DISCUSSION THREAD

**Discussion by hughbris** [getgrav/grav#3545] [Help] [answered]
I hope I'm simply overlooking something in the docs here. I want to create a simple YAML structure using admin blueprints. It's a page blueprint, looking for an ingredients list in frontmatter like:

```yaml
ingredients:
  - "1 tbsp sugar"
  - "2 lemons"
  - "4 cups of wholemeal flour"
```

It's most similar to the `array` type, but without keys, or a `list` type with a single unlabeled child field. I've tried variations of these without success. It seems strange to me if this isn't a supported frontmatter structure.

Anyone know how I can achieve this? I know I can make a custom type. If anyone has done it this way, could you please provide code?

Many thanks.

**Comment by rhukster** [comment, marked as answer]
Use an array type with `value_only: true` in the definition (same level as `type: array`

**Comment by hughbris** [comment]
Superb, thank you! I might add that description to the docs, though it's reasonably self-explanatory and I did overlook it!
